packages/forms-web-app/src/routes/file-based-router/comment-planning-appeal/your-name/controller.js

```diff
@@ -0,0 +1,77 @@
+/** @type {import('express').RequestHandler} */
+const yourNameGet = (req, res) => {
+	if (!req.session.interestedParty?.appealNumber) {
+		return res.redirect(`enter-appeal-reference`);
+	}
+
+	const interestedParty = req.session.interestedParty || {};
```

Comment:
I don't think you'll ever reach `|| {}` since you've got `if (!req.session.interestedParty?.appealNumber)` above this

Reply:
Good point